preview/app/views/components/icon/_arguments.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="max-w-screen-md font-sans font-light">
-  <p class="text-p2 text-cool-black mb-32">Display icons. Uses <code class="pi-inline-code">&lt;use&gt;</code> to insert SVG's. Assumes the spritesheet is already present on the page (this can be done with <code class="pi-inline-code">core/load-sprites.js</code>). Takes the following arguments:</p>
+  <p class="text-p2 text-cool-black mb-32">Display icons. Uses <code class="ui-inline-code">&lt;use&gt;</code> to insert SVG's. Assumes the spritesheet is already present on the page (this can be done with <code class="ui-inline-code">core/load-sprites.js</code>). Takes the following arguments:</p>
```

Comment:
the new css class is `ui-textarea`, but the class being used here is `ui-inline-code`

Am I missing some context?

Reply:
Nope, just fixed that along adding the class. I'll make that more obvious by splitting the commits